File: C/EX15.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  float peso, altura, c;
  
  printf("Digite o seu peso: ");
  scanf("%f", &peso);
 
  printf("Digite a sua altura: ");
  scanf("%f", &altura);
 
  c = peso / (altura*altura);
 
  if (c < 20)
    {
      printf("Abaixo do peso!");
    }
         else if(c < 25)
        {
         printf("Peso ideal");
        }
            else
            {
            printf("Acima do peso!");
            }
 
  return 0;
}

File: Python/Ex15.py
p = float(input("Digite o valor do peso: "))
a = float(input("Digite o valor da altura: "))

c = p/(a*a)


if (c < 20):
    print ("Você está abaixo do peso")

elif c < 25:
    print ("Você está com um peso ideal")
else:
    print ("Você está acima do peso")

